docs/api_specs/includes/watcher_api_specs.md

```diff
@@ -0,0 +1,691 @@
+
+### Request Body
+
+Attribute | Type | Description
+--------- | ------- | -----------
+transaction | Hex encoded string | Signed transaction RLP-encoded to bytes and HEX-encoded to string
+
+
+
+
+## Status
+
+```shell
+curl -X POST http://localhost:4000/status
+```
+
+```elixir
+// TODO
+```
+
+```javascript
+// TODO
+```
+
+> The above command returns JSON document:
+
+```json
+{
+    "version": "1",
+    "success": true,
+    "data": {
+        "last_validated_child_block_number": 10000,
+        "last_mined_child_block_timestamp": 1535031020,
+        "last_mined_child_block_number": 11000,
+        "eth_syncing": true,
+        "byzantine_events":
+        [
+            {
+                "event": "invalid_exit",
+                "details": {
+                    "eth_height"  : 100,
+                    "utxo_pos"  : 10001001,
+                    "owner"  : "B3256026863EB6AE5B06FA396AB09069784EA8EA",
+                    "currency"  : "0000000000000000000000000000000000000000",
+                    "amount" : 100
+                }
+            }
+        ]
+    }
+}
+```
+
+Gets plasma network and Watcher status
+
+### HTTP Request
+
+`POST /status`
+
+### Request Body
+
+No parameters are required.
+
+<aside class="warning">
+The most critical function of the Watcher is to monitor the ChildChain and report dishonest activity. 
+The user must call the `/status` endpoint periodically to check. Any situation that requires the user to either exit or challenge an invalid exit will be included in the `byzantine_events` field.
+</aside>
+
+### Byzantine events
+All of the following events indicate byzantine behaviour and that the user should either exit or challenge.
+
+#### `invalid_exit`
+> An invalid_exit event
+
+```json
+{
+    "event": "invalid_exit",
+    "details": {
+        "eth_height"  : 3521678,
+        "utxo_pos"  : 10001001,
+        "owner"  : "B3256026863EB6AE5B06FA396AB09069784EA8EA",
+        "currency"  : "0000000000000000000000000000000000000000",
+        "amount" : 100
+    }
+}
+```
+
+Indicates that an invalid exit is occurring. It should be challenged.
+
+
+#### `unchallenged_exit`
+> An unchallenged_exit event
+
+```json
+{
+    "event": "unchallenged_exit",
+    "details": {
+        "eth_height"  : 3521678,
+        "utxo_pos"  : 10001001,
+        "owner"  : "B3256026863EB6AE5B06FA396AB09069784EA8EA",
+        "currency"  : "0000000000000000000000000000000000000000",
+        "amount" : 100
+    }
+}
+```
+
+Indicates that an invalid exit is dangerously close to finalization and hasn't been challenged. User should exit.
+
+
+#### `invalid_block`
+> An invalid_block event
+
+```json
+{
+    "event": "invalid_block",
+    "details": {
+        "blockhash"  : "DB32876CC6F26E96B9291682F3AF4A04C2AA2269747839F14F1A8C529CF90225",
+        "blocknum"  : 10000,
+    }
+}
+```
+
+A block containing an invalid tx has been processed [are there other reasons a block can be invalid?]. User should exit.
```

Comment:
re the question here: yes
 - invalid merkle root
 - totally corrupt, malformed data, that can't be decoded